homeassistant/components/melcloud/climate.py

```diff
@@ -73,6 +83,18 @@
         True,
     )
 
+    platform = entity_platform.current_platform.get()
+    platform.async_register_entity_service(
+        "set_vane_horizontal",
+        {vol.Required("position"): cv.string},
+        "async_set_vane_horizontal",
+    )
+    platform.async_register_entity_service(
+        "set_vane_vertical",
+        {vol.Required("entity_id"): cv.entity_id, vol.Required("position"): cv.string},
```

Comment:
Entity Id is not used, please remove it.

Reply:
Done.